### DISCUSSION TITLE
A plan to upgrade action to v3

### DISCUSSION THREAD

**Discussion by lyricat** [pandodao/developers.pando.im#19] [Announcements]
- Effective Date: `2023-07-30`
- Affected Protocols: all protocols

We plan to add a new field `checksum` to the action, which is used to verify the integrity of the action.

Please read the draft [here](https://developers.pando.im/references/action.html#pando-action-protocol-checksum)

**Comment by coolit** [comment]
I use Golang sdk to generate memo only, in this case, do I need to do anything? Thanks.
